CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -358,7 +358,7 @@ public static function formRule(array $fields, $files, $self) {
         'target_id' => 'recipient_manual_id',
       ],
     ];
-    if ($fields['limit_to'] != '' && array_key_exists($fields['recipient'], $recipientKind) && empty($fields[$recipientKind[$fields['recipient']]['target_id']])) {
+    if ($fields['limit_to'] != 1 && array_key_exists($fields['recipient'], $recipientKind) && empty($fields[$recipientKind[$fields['recipient']]['target_id']])) {
       $errors[$recipientKind[$fields['recipient']]['target_id']] = ts('If "Also include" or "Limit to" are selected, you must specify at least one %1', [1 => $recipientKind[$fields['recipient']]['name']]);
     }
 
```

Comment:
From https://lab.civicrm.org/dev/core/-/issues/3464#note_74627

If the limit_to value is 1 then remove the value of the Group ID field on save of this form.